### DISCUSSION TITLE
能否对pve添加原生支持呢

### DISCUSSION THREAD

**Discussion by ifwwwx** [lollipopkit/flutter_server_box#289] [Ideas]
serverbox搭配服务器体验相当的棒，希望能对pve这种虚拟化系统增加原生支持，实现类似proxmon连接pve的那种效果就更好了
![Screenshot_2024-02-28-11-42-10-633_dev reimu proxmon](https://github.com/lollipopkit/flutter_server_box/assets/65443757/f67e21d9-bb46-482e-a6b0-2bb66af558a7)

**Comment by ElementBridge** [comment]
支持，PVE太缺移动端app了，proxmox好几年不更新了

**Comment by lollipopkit** [comment]
Added to TodoList

https://github.com/lollipopkit/flutter_server_box/issues/307

**Comment by gfgjs** [comment]
请问你的截图是什么软件

**Reply by ifwwwx** [reply]
proxmon,目前已经停更很久，安卓14且play商店已经无法安装，https://ifwww.com/posts/61099.html#pve-%E4%BD%BF%E7%94%A8%E7%AC%94%E8%AE%B0

**Reply by ifwwwx** [reply]
链接是安装包

**Comment by GT-610** [comment]
已添加。
